platform-plugins.md

```diff
@@ -246,7 +246,7 @@ Next,
     }
     ```
 1. Replace the implementation in `HelloPlugin.m` with a forwarding call:
-    ```objective-c
+    ```objc
```

Comment:
I think it's `objectivec`? See https://github.com/flutter/website/blob/master/platform-channels.md